src/lib/resource/sections/applicant-non-sa-infections.js

```diff
@@ -83,19 +83,9 @@ module.exports = {
     },
     route: {
         on: {
-            ANSWER: [
+            ANSWER:  [
```

Comment:
Linter should have removed this, no?

Reply:
![image](https://user-images.githubusercontent.com/41323244/141812988-5a094334-519a-4c1a-aac6-699b9e7dad9c.png)
cheers, just added this config to vs code.